scripts/operations/node_management/Add_River_Cabinets/update_ncn_etc_hosts.py

```diff
@@ -0,0 +1,250 @@
+#! /usr/bin/env python3
+# MIT License
+#
+# (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+import argparse
+import datetime
+import json
+import http
+import os
+import requests
+import subprocess
+import sys
+import urllib3
+
+SLS_URL = None
+
+# TODO need to create a common library between this script and the add_management_ncn.py script
```

Comment:
Do we want to commit this TODO comment into the customer-facing code itself that we're shipping? Would this be better to record in a Jira, since we're just outlining work that ought to be done?

Reply:
Opened: CASMHMS-5716

Reply:
https://github.com/Cray-HPE/docs-csm/pull/2344/commits/c0fb7b04a18b9ef5b38729cf25632254bdbc4eb7